cms/djangoapps/contentstore/proctoring.py

```diff
@@ -30,16 +30,6 @@ def register_proctored_exams(course_key):
     subsystem. Likewise, if formerly registered exams are unmarked, then those
     registered exams are marked as inactive
     """
-
```

Comment:
@chrisndodge said that timed exams are still gated by the ENABLE_PROCTORED_EXAMS feature flag. So why was this code removed?